### PR TITLE
embryonic Slack notification for master builds

### PR DESCRIPTION
**Pull request by garyverhaegen-da**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -86,3 +98,26 @@ jobs:
...
           addChangeLog: false
           isPrerelease: true
+      - bash: |
+          set -eux pipefail
+          echo "##vso[task.setvariable variable=status;isOutput=true]$(Agent.JobStatus)"
+        name: status
+
+  - job: notify
+    dependsOn: ["linux", "macos", "windows", "release"]
+    condition: always() #eq(variables['Build.SourceBranchName'], 'master')
```
Comment (cocreature) [inline azure-pipelines.yml]: I guess you only disabled this for testing?

Comment (garyverhaegen-da) [reply azure-pipelines.yml]: Yes, will definitely re-enable before merging.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -25,6 +25,14 @@ jobs:
       - template: ci/build-unix.yml
         parameters:
           name: linux
+      - bash: |
```
Comment (flokli) [inline azure-pipelines.yml]: hoping azures' yaml parser supports this: can we use [anchors and merge keys](https://camel.readthedocs.io/en/latest/yamlref.html#merge-keys) to reduce code duplication?

Comment (garyverhaegen-da) [reply azure-pipelines.yml]: > /azure-pipelines.yml: Anchors are not currently supported. Remove the anchor 'SLACK'

Nope :(

Comment (flokli) [reply azure-pipelines.yml]: Meh :-/

https://github.com/Microsoft/azure-pipelines-yaml/issues/187

**Pull request merged**